doc/docs/modules/ROOT/pages/writing.adoc

```diff
@@ -780,4 +774,40 @@ MyNodeWithFlattenedMap {
     time: 1,
     `table.key.inner.key`: 'innerValue1' // but it could be `value1` as the order is not guaranteed
 }
-----
\ No newline at end of file
+----
+
+=== Group duplicated keys to array of values
+
+You can use the option `schema.map.group.duplicate.keys` to avoid this problem, what the connector will do is to group all and only the values with duplicate keys into an array with all the values. The default value for the option is `false`.
```

Comment:
```suggestion
You can use the option `schema.map.group.duplicate.keys` to avoid this problem. The connector will group all the values with the same keys into an array. The default value for the option is `false`.
```